lkcrypto.c

```diff
@@ -209,7 +276,104 @@ static int decrypt_encrypt(lua_State *L, int dec_enc)
 	} else {
 		kfree(buffer);
 		return luaL_error(L, "%s failed %d",
-				  dec_enc ? "encrypt" : "decrypt");
+				  operation ? "encrypt" : "decrypt");
```

Comment:
Missing parameter for the `%d` (as I see was the case already before, whoops), and again maybe use `operation == ENCODING` in the ternary.